web/styles/settings.css

```diff
@@ -550,6 +550,15 @@ input[type="checkbox"] {
     }
 }
 
+.realm_enter_sends kbd {
+    opacity: 0.9;
+    font-size: 11px;
+    color: hsl(0deg 0% 50%);
+    position: relative;
+    bottom: 1px;
+    margin: 0 2px;
+}
+
```

Comment:
We need to override the default css for `kbd` element in `app_component.css` as the default css makes the `kbd` element huge in size. 

Checkout this [CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/user.20research.3A.20Enter.20sends.20setting/near/1822886) discussion for more context.